### PR TITLE
#182 - Mark new co-organizers as admins

### LINKED ISSUE #182: [ProductionData] Update list of organizers
January 2018 brought some organizational changes to ZgPHP meetup - our main organizer stepped down, and group now has 4 new (additional) organizers.
They all should be added to admin list, so they can be excluded from raffle.

### PR DESCRIPTION
**Pull request by HcsOmot**
New co-organizers should be excluded from the raffle, so we need to mark
them as admin in the list of users.

Closes #182

### CODE REVIEW

**Comment by HcsOmot** [comment]
Aaaaaand Behat tests are failing again.

**Review by msvrtan — APPROVED** [review]

**Comment by msvrtan** [comment]
lets fix that later ;)

File: src/Migrations/Version20180126123008.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        // this up() migration promotes four records as ZgPhp Meetup organizers
+        $this->addSql('UPDATE joindinUsers SET organizer = TRUE WHERE id IN (27939, 31316, 47268, 26764)');
+    }
+
+    public function down(Schema $schema)
+    {
+        // this down() migration demotes four records representing ZgPhp Meetup organizers
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(),
+            'Migration can only be executed safely on \'postgresql\'.');
+        $this->addSql('UPDATE joindinUsers SET organizer = FALSE WHERE id IN (27939, 31316, 47268, 26764)');
```
Comment (msvrtan) [inline src/Migrations/Version20180126123008.php:26]: JAOOOOO @HcsOmot ... nije FALSE nego TRUE!!!

Bas smo posteno sljepi :)

**Comment by HcsOmot** [comment]
Wat? Zašto true? Down migration?

**Comment by HcsOmot** [comment]
Wat? Zašto true? Down migration?

**Comment by msvrtan** [comment]
uh, slijep sam :(

**Comment by msvrtan** [comment]
Ali uglavnom, organizatori su se pojavili na raffleu u veljaci ... tako da nes nije kak treba :(

**Comment by HcsOmot** [comment]
I'll look into it third thing tomorrow!

**Comment by msvrtan** [comment]
bem mu misa, sad se ne pojavljuju u listi ....

**Comment by msvrtan** [comment]
zanemari, sada radi ok ... naime Zoka je izvukao jednoga od vas u veljaci , sjetih se toga i vidim da je davno namergano ...

**Comment by HcsOmot** [comment]
Sori, ne pada mi ništa na pamet trenutno. Pogledam sutra. A ionako već dosta kasni razvoj. My bad...

**Pull request merged**